site/hxapi/hotel/bkg/index.md

```diff
@@ -2,205 +2,347 @@
 
 ---
 
-# Hotel Booking
+## Hotel Booking Request
 
-As with all bookings, this must be a POST and it must be made over HTTPS. Intermediaries will be sending credit card information, Agents will send their login details so HTTPS is required in both instances.
+### Method
 
-If you do not send the ABTANumber field then your booking will not be tracked to your agent code. For agent bookings, the Password field is also required, intermediaries do not have a password.
+POST
 
-To retrieve all details about the booking, perform a [GET request to view the booking](/hxapi/viewamendcancel) at the location specified in `<MoreInfoURL>`.
+### Endpoint
 
-Currently, we can only accept 2 rooms per booking. If the customer needs more, you will need to make multiple requests. If an intermediary account, this would lead to multiple transactions on the credit card, and multiple charges, therefore we advise against it. Agents accounts do not suffer from this, they can make multiple requests as they process the payment themselves.
+The endpoint to use is (where "foo" is the hotel code):
 
-## /hotel/foo
+```
+https://api.holidayextras.co.uk/v1/hotel/foo
+```
 
-where foo is the selected hotel code
+For example, for _Mercure at London Heathrow_ the endpoint is:
 
-e.g. /hotel/LHRSOF
+```
+https://api.holidayextras.co.uk/v1/hotel/LHRMEA
+```
 
-### Method
+### Request Parameters
+
+The parameters _must_ be sent in the body of the request, as `x-www-form-urlencoded` data.
+
+NB: All parameter names are case sensitive.
+
+ | Name        | Data Type    | Format | Mandatory? | Additional Information |
+ | ----        | ----    | --------- | -------- | ---------------------- |
+ | ABTANumber  | String  | [A-Z0-9] 5 chars | Y | This is also known as an 'agent code'. <br>This will be confirmed to you by your Account Manager during set up.|
```

Comment:
With markdown you are not writing formatted output, just structured text. Due to this there are certain formatting you need to give up as it is not supported by default, newlines in tables seem to be one of these. The only alternative is to write HTML into the markdown which defeats the point of using such as language.